scripts/update-data-yaml.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
```

Comment:
I believe this file isn't executable by default, can you run chmod +x and commit it again?